docs/administration-stats.md

```diff
@@ -22,8 +22,8 @@ All stats below are **reset** to 0 upon broker restart or topic unloading, **exc
 | msgRateOut                                       | The sum of all local and replication consumers' dispatch rates (message per second).                                                                                                                                                            |
 | msgThroughputOut                                 | The sum of all local and replication consumers' dispatch rates (byte per second).                                                                                                                                                               |
 | averageMsgSize                                   | The average size (bytes) of messages published within the last interval.                                                                                                                                                                        |
-| storageSize*                                     | The sum of the ledgers' storage size **in BookKeeper** for a topic (in bytes). <br/><br/>**Note**: the `total storage size of a topic` = `storageSize` + `offloadedStorageSize`.                                                                |
-| offloadedStorageSize*                            | The sum of the storage size **in tiered storage** for a topic (in bytes).<br/><br/>**Note**: the `total storage size of a topic` = `storageSize` + `offloadedStorageSize`.                                                                      |
+| storageSize*                                     | The sum of the ledgers' storage size **in BookKeeper** and **in tiered storage** for a topic (in bytes). <br/><br/>**Note**: the `total storage size of a topic` = `storageSize`.                                                               |
```

Comment:
Maybe we don't need the `**Note**` anymore.

Reply:
Yes, for storageSize, we don't need it anymore